fast64_internal/utility.py

```diff
@@ -523,6 +525,118 @@ def deleteIfFound(filePath, stringValue):
 			fileData.write(stringData)
 		fileData.close()
 
+def yield_children(obj: bpy.types.Object):
+	yield obj
+	if obj.children:
+		for o in obj.children:
+			yield from yield_children(o)
+
+def iter_from_active():
+	active_obj = bpy.context.view_layer.objects.active # keep original obj reference
+	yield from yield_children(active_obj)
+	bpy.context.view_layer.objects.active = active_obj
```

Comment:
that's pretty weird behavior imo,
I would make code calling `iter_from_active` responsible for preserving the context (active object),
rather than doing it in the generator function and implicitly restore context once iteration is done